com.b2international.snowowl.validation.snomed/src/main/resources/scripts/ruleSnomedCommon1.groovy

```diff
@@ -0,0 +1,66 @@
+import com.b2international.index.aggregations.Aggregation
+import com.b2international.index.aggregations.AggregationBuilder
+import com.b2international.index.aggregations.Bucket
+import com.b2international.index.query.Expression
+import java.util.stream.Collectors
+import com.b2international.index.revision.RevisionSearcher
+import com.b2international.snowowl.core.ComponentIdentifier
+import com.b2international.snowowl.snomed.common.SnomedTerminologyComponentConstants
+import com.b2international.snowowl.snomed.core.domain.SnomedConcepts
+import com.b2international.snowowl.snomed.core.domain.SnomedRelationships
+import com.b2international.snowowl.snomed.datastore.index.entry.SnomedRelationshipIndexEntry
+import com.b2international.snowowl.snomed.datastore.request.SnomedRequests
+import com.b2international.snowowl.core.domain.IComponent
+import com.google.common.collect.Sets
+
+Set<ComponentIdentifier> invalidIds = Sets.newHashSet()
+
+SnomedConcepts inactiveConcepts = SnomedRequests.prepareSearchConcept()
```

Comment:
Just to save a bunch of memory, return only the IDs of the matching components. In order to that efficiently, use the Index API directly rather than the high-level request API.